code/game/objects/items/dehy_carp.dm

```diff
@@ -7,6 +7,7 @@
 /obj/item/toy/carpplushie/dehy_carp
 	var/mob/owner = null	//Carp doesn't attack owner, set when using in hand
 	var/owned = 0	//Boolean, no owner to begin with
+	var/mobtype = /mob/living/simple_animal/hostile/carp //So admins can change what mob spawns via var fuckery
```

Comment:
that seems like it would add unnecessary bloat. i only made this feature for admin abuse that would be rare at best and i still dont know why its in this pr